go.mod

```diff
@@ -1,6 +1,6 @@
 module go.k6.io/k6
 
-go 1.16
+go 1.17
```

Comment:
```suggestion
go 1.16
```

hmm maybe don't update this quite yet? :sweat_smile:

Reply:
Ok, i will fix it soon. :)